css/main.css

```diff
@@ -166,7 +165,7 @@ h3 {
 }
 
 .accordion-header.sticky-top {
-	top: 180px;
+	top: 110px;
```

Comment:
what does this change have to do with the logo?

Reply:
The logo not being as tall has made the header shorter. This left empty space between the green header and the top accordion header.